resource/css/rtl.css

```diff
@@ -0,0 +1,700 @@
+}
+
+html[dir="rtl"] #search-field {
+  padding-right: 8px;
+  padding-left: 22px;
+}
+
+html[dir="rtl"] .qtip-skosmos {
+  margin-right: 9px;
+  margin-left: initial;
+  direction: rtl;
+}
+
+html[dir="rtl"] .reified-property-value {
+  float: right;
+}
+
+
+html[dir="rtl"] span.xl-label > img {
+  margin: -2px 0 0 2px;
+}
+
+/* front page stuff
+ ***************************/  
+
+ html[dir="rtl"] .right-box {
+  left: 0;
+  right: initial;
+}
+
+html[dir="rtl"] .vocabulary-separator {
+  margin-right: 28%;
+  margin-left: initial;
+}
+
+html[dir="rtl"] #vocabulary-list > div > h2, #vocabulary-list-wide > div > h2 {
+  padding: 15px 10px 15px 0;
+}
+
+html[dir="rtl"] .vocab-category {
+  padding-right: 0;
+  padding-left: initial;
+}
+
+html[dir="rtl"] .vocab-category > h2 {
+  margin: 0 0 5px 0;
+}
+
+html[dir="rtl"] .vocab-category > ul {
+  float: left;
+  padding-right: 0;
+}
+
+html[dir="rtl"] .vocab-category > li {
+  float: right;
+}
+
+/* feedback page
+ ***************************/
+html[dir="rtl"] #feedback-vocid {
+  text-align: right;
+}
+
+html[dir="rtl"] #feedback-vocid > .caret {
+  left: 10px;
+  right: initial;
+}
+
+html[dir="rtl"] .feedback-logo {
+  right: 200px;
+  left: initial;
+}
+
+html[dir="rtl"] .feedback-box {
+  float: right;
+}
+
+html[dir="rtl"] #send-feedback {
+  float:left;
+  margin: 5px 0 15px 0;
+}
+
+/* about page
+ ***************************/
+
+html[dir="rtl"] .special-container {
+  right: 0;
+  margin-right: 200px;
+  margin-left: initial;
+}
+
+/* sidebar styles
+ ***************************/
+
+ html[dir="rtl"] .sidebar-grey ul, .sidebar-grey p {
+  padding-right: 0;
+  padding-left: initial;
+}
+
+html[dir="rtl"] .multiselect-container li > a > label.checkbox {
+  padding: 3px 25px 3px 5px;
+}
+
+html[dir="rtl"] .multiselect-container li > a > label.checkbox > input {
+  margin-right:-20px;
+  margin-left: initial;
+}
+
+html[dir="rtl"] .search-options .multiselect > span {
+  float: right;
+  text-align: right;
+}
+
+html[dir="rtl"] .search-options .multiselect > .caret {
+  float: left;
+  left: 5px;
+  right:initial;
+}
+
+html[dir="rtl"] div#sidebar-grey > div > div > form.search-options {
+  margin-left: 0;
+  margin-right: initial;
+}
+
+html[dir="rtl"] .search-options {
+  padding-right: 15px;
+  padding-left: initial;
+}
+
+html[dir="rtl"] .search-result-listing .btn-default {
+  left: 10px;
+}
+
+html[dir="rtl"] .search-options .mCSB_container > li > a > .radio {
+  margin: 0;
+  padding-right: 25px;
+  padding-left: initial;
+}
+
+html[dir="rtl"] .search-options .mCSB_container {
+  text-align: right;
+}
+
+html[dir="rtl"] #alphabetical-menu {
+  margin: 15px 15px 0 0;
+}
+
+html[dir="rtl"] .nav {
+  padding-right:0;
+}
+
+html[dir="rtl"] .group-container {
+  margin: 15px 15px 5px 0;
+}
+
+html[dir="rtl"] li.sub-group {
+  margin-right: 25px;
+  margin-left: initial;
+}
+
+/* scrollbars
+ ****************************/
+html[dir="rtl"] .mCSB_container {
+  margin-left: 15px !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/NatLibFi/Skosmos/pullRequest?prid=4951454)